freqtrade/persistence/trade_model.py

```diff
@@ -468,6 +469,23 @@ def safe_quote_currency(self) -> str:
         except IndexError:
             return ''
 
+    @hybrid_property
+    def open_orders_count(self) -> int:
+        return len(self.open_orders)
+
+    @hybrid_property
```

Comment:
what's the reason to use `@hybrid_property` over `@property`?

in my understanding, the "hybrid" is needed to define a property that works both at instance and class level - which doesn't seem to make sense here (what would `Trade.open_orders_count` even mean?

Reply:
You are right will change that